packages/lib/server/repository/organization.ts

```diff
@@ -90,6 +90,7 @@ export class OrganizationRepository {
       email: owner.email,
       username: ownerUsernameInOrg,
       organizationId: organization.id,
+      locked: false,
```

Comment:
We'll check the new org owner against the watch list in this PR https://github.com/calcom/cal.com/pull/19201